rails_event_store_active_record/lib/rails_event_store_active_record/generators/templates/create_event_store_events_template.rb

```diff
@@ -18,25 +16,18 @@ def change
     add_index :event_store_events_in_streams, [:created_at]
     add_index :event_store_events_in_streams, [:stream, :event_id], unique: true
 
-    if postgres
-      create_table(:event_store_events, id: :uuid, default: 'gen_random_uuid()', force: false) do |t|
-        t.string      :event_type,  null: false
-        t.binary      :metadata
-        t.binary      :data,        null: false
-        t.datetime    :created_at,  null: false
-      end
-    else
-      create_table(:event_store_events, id: false, force: false) do |t|
-        t.string :id, limit: 36, primary_key: true, null: false
-        t.string      :event_type,  null: false
-        t.binary      :metadata
-        t.binary      :data,        null: false
-        t.datetime    :created_at,  null: false
-      end
-      if sqlite && rails_42
-        add_index :event_store_events, :id, unique: true
+    create_table(:event_store_events, force: false) do |t|
+      if postgres
+        t.references :event, null: false, type: :uuid
+      else
+        t.references :event, null: false, type: :string, limit: 36
       end
+      t.string      :event_type,  null: false
+      t.binary      :metadata
+      t.binary      :data,        null: false
+      t.datetime    :created_at,  null: false
     end
+    add_index :event_store_events, :event_id, unique: true
     add_index :event_store_events, :created_at
     add_index :event_store_events, :event_type
```

Comment:
Not sure if in scope but It might be more useful to have `:event_type, :id` than just `:event_type`. Maybe...

Reply:
What benefits do you see of specifying  add_index `:event_store_events, [:event_type, :id]`?